langchain/vectorstores/azuresearch.py

```diff
@@ -195,6 +195,8 @@ def add_texts(
         self,
         texts: Iterable[str],
         metadatas: Optional[List[dict]] = None,
+        extra_fields: Optional[Dict[str, str]] = None,
+        """Allows adding extra fields as a list of dictionaries, which can be retrieved or filtered """
```

Comment:
this isnt usually where docstrings for arguments go, instead we should add to the docstring on line 202

Reply:
Changed docstrings accordingly